CONTRIBUTING.md

```diff
@@ -18,6 +18,9 @@ You need a basic understanding of [Git and GitHub.com](https://guides.github.com
 
 For an example where this process led to publication of a new article, see [Issue &num;67](https://github.com/dotnet/docs/issues/67) and [Pull Request &num;798](https://github.com/dotnet/docs/pull/798) in the .NET Docs repository. The new article is [Documenting your code](https://docs.microsoft.com/dotnet/articles/csharp/codedoc).
 
+## Using the Docs Authoring Pack extension in Visual Studio Code 
+If you use Visual Studio Code to contribute to the ASP.NET documentation, you can boost your productivity by installing the [Docs Authoring Pack](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack) extension. The extension provides a variety of tools that helps with Markdown linting, code spell checking, and article templates.
```

Comment:
Quick nits:

* Add a blank line between the section heading and the text.
* Say "Use" instead of "Using" in the section heading (there's an attempt to at least avoid gerunds in section headings). Alternatively, you could just drop it and go with something like: **Docs Authoring Pack extension in Visual Studio Code**

Reply:
I pushed another update with the suggested changes. Thanks, @guardrex.